applications/object_detection_torch/package-app.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -e
+
+GIT_ROOT=$(readlink -f ./$(git rev-parse --show-cdup))
+
+# Utilities
+
+YELLOW="\e[1;33m"
+RED="\e[1;31m"
+NOCOLOR="\e[0m"
+
+print_error() {
+    echo -e "${RED}ERROR${NOCOLOR}:" $*
+}
+
+get_host_gpu() {
+    if ! command -v nvidia-smi >/dev/null; then
+        print_error Y "Could not find any GPU drivers on host. Defaulting build to target dGPU/CPU stack."
+        echo -n "dgpu"
+    elif nvidia-smi  2>/dev/null | grep nvgpu -q; then
+        echo -n "igpu"
+    else
+        echo -n "dgpu"
+    fi
+}
+
+get_host_arch() {
+    echo -n "$(uname -m)"
+}
+
+if [ ! -d $GIT_ROOT/build/object_detection_torch ]; then
+    print_error "Please build the Object Detection Torch application first with the following command:"
+    print_error "./dev_container build_and_run object_detection_torch"
+    exit -1
+fi
+
+APP_PATH="$GIT_ROOT/object_detection_torch"
+echo Creating application directory $APP_PATH...
+mkdir -p $APP_PATH
+echo Copying application files to $APP_PATH...
+cp -f $GIT_ROOT/build/object_detection_torch/applications/object_detection_torch/object_detection_torch $APP_PATH
+cp -f $GIT_ROOT/build/object_detection_torch/applications/object_detection_torch/object_detection_torch.yaml $APP_PATH
+
+PLATFORM=x64-workstation
+GPU=$(get_host_gpu)
+if [ $(get_host_arch) == "aarch64" ]; then
+    PLATFORM=igx-orin-devkit
+fi
+
+echo -e "done\n"
+echo -e Use the following commands to package and run the Object Detection Torch application:
```

Comment:
Why print these commands rather than running them directly in the bash script?

Reply:
When users install the PyPI version of the HSDK in a virtual environment, the CLI may not be accessible in a Holohub repo. Therefore, instead of running it automatically, the commands are printed.

Reply:
I'm not familiar, what additional step would the user need to have access to the CLI? Could we include those steps in the printout? Under current behavior it's not clear that those commands may not be immediately valid.

Reply:
I have updated the message with a link to get more info.

Reply:
That's doable, and we can always attempt to automate calling and fallback to printing the commands. I created this PR to reduce the number of steps for QA to follow in the test procedures, and I did not want to automate every step as
1) I believe printing the commands and arguments gives the users an excellent example of using the CLI because there are just too many options
2) in case users install the PyPI package in their virtual environment, then automating this won't work

If having automated solutions is necessary, I can create a separate issue and work on it later.

Reply:
Okay, understood, using this as a minimum configuration for SQA while still exposing users to available options makes sense. Would be nice to explore automating and consolidating scripts, but OK if it's a low priority. Thanks for clarifying, good to leave as is for this effort.